packages/vue/src/Form/DatePicker/DatePicker.vue

```diff
@@ -72,7 +72,7 @@ const props = defineProps({
   },
   isIndefinite: {
     type: Boolean,
-    default: true
+    default: false
```

Comment:
This default value will apply in all datepicker in the project where `isIndefinite` is not set. Is this expected?

Reply:
yes.
that is why I changed it to false. 
if it is true, it will always look like below:

<img width="320" alt="image" src="https://github.com/hit-pay/orchid/assets/28958299/8bb51135-eaa9-4711-ada5-92016a675679">

but it should be optional, to add it or not.